docs/source/download.rst

```diff
@@ -141,177 +177,42 @@ Otherwise, use the appropriate values of ``PETSC_DIR`` and ``PETSC_ARCH``.
    It is not necessary that h5py be built with MPI support, although
    Firedrake supports both options.
 
+Further dependencies
+~~~~~~~~~~~~~~~~~~~~
 
-Potential installation errors on Mac OS
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Firedrake depends on the Python packages PyOP2_, FFC_, FIAT_ and UFL_.
```

Comment:
My intention is that that link goes to the pyop2 installation instructions, which tell you how to get COFFEE.